skype/skype-ps/skype/New-CsTeamsAppSetupPolicy.md

```diff
@@ -37,17 +37,196 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
 ## EXAMPLES
 
 ### Example 1
-Intentionally not provided
+
+Create a new TeamsAppSetupPolicy,if no parameters are specified, the Global Policy configuration is used by default.
+```powershell
+$identity = "New-TestTeamsAppSetupPolicy" + (Get-Date -Format FileDateTimeUniversal)
+New-CsTeamsAppSetupPolicy -Identity $identity
+```
+
+### Example 2
+
+Create a new TeamsAppSetupPolicy.Set users can upload a custom app package in the Teams app because set AllowSideLoading as true,and set users’s existing app pins can be added to the list of pinned apps set because set AllowUserPinning as true.
```

Comment:
```suggestion
Create a new TeamsAppSetupPolicy. Users can upload a custom app package in the Teams app because AllowSideLoading is set as True, and existing app pins can be added to the list of pinned apps because AllowUserPinning is set as True.

```